src/devices/Bmp180/Bmp180.cs

```diff
@@ -226,7 +237,8 @@ internal ushort Read16BitsFromRegisterBE(byte register)
 
             return BinaryPrimitives.ReadUInt16BigEndian(bytes);            
         }
-        
+
+        /// <inheritdoc/>

```

Comment:
@krwq keep in mind that the porting tool currently does not know how to handle this tag.

Reply:
These won't be ported.